packages/iobio-react-components/src/histogram.tsx

```diff
@@ -26,12 +26,12 @@ function IobioHistogram({
 	brokerKey,
 	styles,
 	ignoreOutliers = false,
-	title,
+	label,
```

Comment:
it may seem like nitpicking, but it's highly recommended to keep properties alphabetised whenever possible.

not a change required for this PR, but something to keep in mind for future changes.

Reply:
I generally try to do that, guess I overlooked this one

Reply:
Updated

Reply:
here and in a few other places. don't worry too much about it for now.
it's more DevX than functionality, so it's not an urgent thing to address.